components/runtime/openjdk-8/Makefile

```diff
@@ -0,0 +1,114 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Alexander Pyhalov
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= oppenjdk-8
+
+OPENJDK_BUILD_NUMBER=92
+COMPONENT_VERSION=1.8.$(OPENJDK_BUILD_NUMBER)
+COMPONENT_SUMMARY= Open-source implementation of the Java Platform, Standard Edition
+COMPONENT_SRC= openjdk
+COMPONENT_ARCHIVE= openjdk-$(COMPONENT_VERSION)-20160506.tar.xz
+COMPONENT_ARCHIVE_HASH= \
+  sha256:dbf98bea04fba78a625c4feb3417a3669105e2099f59829abd71894ebea4a684
+COMPONENT_ARCHIVE_URL= \
+  http://ftp.netbsd.org/pub/pkgsrc/distfiles/LOCAL_PORTS/openjdk7/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL = http://openjdk.java.net/
+
+CLEAN_PATHS += $(BUILD_DIR)
+CLEAN_PATHS += cacerts.out
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/ips.mk
+include $(WS_TOP)/make-rules/configure.mk
+
+PATCH_LEVEL=0
+
+CC=gcc
+CXX=g++
+PATH=$(GCC_ROOT)/bin:/usr/gnu/bin:/usr/bin:/usr/sbin
+
+COMPONENT_PRE_CONFIGURE_ACTION =        (cp -r $(SOURCE_DIR)/* $(@D))
+CONFIGURE_SCRIPT=	$(BUILD_DIR_$(BITS))/common/autoconf/configure
+
+CONFIGURE_OPTIONS+=        --with-boot-jdk=$(JAVA_HOME)
+CONFIGURE_OPTIONS+=        --with-zlib=system
+CONFIGURE_OPTIONS+=        --with-giflib=bundled
+CONFIGURE_OPTIONS+=        --with-cups
+CONFIGURE_OPTIONS+=        --disable-freetype-bundling
+CONFIGURE_OPTIONS+=        --with-memory-size=768
+CONFIGURE_OPTIONS+=        --disable-precompiled-headers
+CONFIGURE_OPTIONS+=        --disable-ccache
+CONFIGURE_OPTIONS+=        --with-user-release-suffix=oi-$(COMPONENT_VERSION)
+CONFIGURE_OPTIONS+=        --with-update-version=$(OPENJDK_BUILD_NUMBER)
+CONFIGURE_OPTIONS+=        --with-build-number=b14
+
+CONFIGURE_ENV+=	AS=/usr/gnu/bin/as
+CONFIGURE_ENV+=	STRIP=/usr/bin/strip
```

Comment:
Consider adding GAS=/usr/gnu/bin/as and STRIP=/usr/bin/strip to shared-macros.mk perhaps.